### DISCUSSION TITLE
можно ли удалить все треки сразу не из плейлиста,а из любимых треков?

### DISCUSSION THREAD

**Discussion by anaitmarty** [Chimildic/YaMuTools#49] [Вопрос/ответ] [answered]
не могу найти такую функцию

**Comment by Chimildic** [comment, marked as answer]
Создайте плейлист из лайков

![image](https://user-images.githubusercontent.com/40138097/164979923-dca0b0c7-526d-476d-b786-e1dca4bfe4e2.png)

Используйте Лайкер

![image](https://user-images.githubusercontent.com/40138097/164979964-0defa62c-0e6f-4cdd-8a27-f2112cf070e6.png)
